package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/flot/flot#readme",
   "devDependencies": {
+    "@babel/polyfill": "^7.4.4",
```

Comment:
add this to include regenerator-runtime node_module

Reply:
As mentioned above, make this a 'dependency'.